Monika After Story/game/script-greetings.rpy

```diff
@@ -3417,3 +3419,255 @@ label greeting_ghost:
+                            m 1rksdlb "It might also ease my mind a bit if you weren't staying up so late in the first place, ahaha..."
+                            m 1eua "I'm just glad you're here now."
+                            m 3tubfu "You'd never be too tired to spend time with me, right?"
+                            m 1hubfb "Ahaha!"
+                        "Maybe...":
+                            m 1dsc "Hmm..."
+                            m 1euc "I wonder what could be causing it?"
+                            m 3eud "You didn't stay up really late last night, did you, [player]?"
+                            m 1ekc "Were you doing something last night?"
+                            m 3rfu "Maybe...{w=2}I don't know..."
+                            m 3tku "Playing a game?"
+                            m 1hub "Ahaha!"
+                            m 1hua "Just teasing you of course~"
+                            m 1tubfb "I hope you didn't mind."
+                            m 1eka "In all seriousness though, I really don't want you neglecting your sleep."
+                            m 2rksdla "It's one thing staying up late just for me..."
+                            m 3rksdla "But leaving and playing another game that late?"
+                            m 1tubfu "Ahaha...I might get a bit jealous, [player]~"
+                            m 1hubfa "But you're here to make up for that now, right?"
+                "No.":
+                    m 1eud "Ah, so I guess you were busy all morning."
+                    m 1eka "I was worred you overslept since you were up so late last night."
+                    m 2rksdla "Especially since you told me you were going to go play another game."
+                    m 1hua "I should have known you'd be responsible and get your sleep though."
+                    m 1hksdlb "You {i}did{/i} get your sleep, right, [player]?"
+                    m 1hub "Ahaha!"
+                    m 1eub "Anyway, now that you're here, we can spend some time together."
+        #SStoMN
+        else:
+            m 2euo "Oh, [player]!"
+            m 1hub "There you are!"
+            m 2hksdlb "Ahaha, sorry... It's just that I haven't seen you all day."
+            m 1rksdla "I kind of expected you to sleep in after staying up so late last night..."
+            m 1rksdlc "But when you didn't even show up all afternoon, I started to miss you a little more than usual."
+            m 2hksdlb "You almost had me worried, ahaha..."
+            m 3tubfb "But you're going to make up all that lost time to me, right?"
+            m 1hubfa "Ehehe, you better~"
+            show monika 5tubfu at t11 zorder MAS_MONIKA_Z with dissolve
+            m 5tubfu "Especially after leaving me for another game last night."
+        $ mas_late_farewell = False
+    #If you didn't stay up late in the first place, normal usage
+    else:
```

Comment:
everything below this else probably could just be elif'd and you can remove this else.